Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonMessageBox.cs

```diff
@@ -118,7 +118,7 @@ public static class KryptonMessageBox
         /// <param name="contentLinkArea">Specifies the area within the <see cref="KryptonLinkWrapLabel"/> to be regarded as a link. See <see cref="LinkArea"/>.</param>
         /// <param name="messageTextAlignment">Specifies how the message text should be aligned. See <see cref="System.Drawing.ContentAlignment"/> for supported values.</param>
         /// <returns>One of the System.Windows.Forms.DialogResult values.</returns>
-        public static DialogResult Show(IWin32Window? owner, string? text, string caption, bool? showCtrlCopy = null,
+        public static DialogResult Show(IWin32Window? owner, string text, string caption, bool? showCtrlCopy = null,
```

Comment:
Why have these been changed. There is a test for passing just a caption and null for the message ?

Reply:
Undo changes